### PR TITLE
Paris SQUAD 107 - Kevin .B

### PR DESCRIPTION
**Pull request by kevbarns**
All queries are commented

### CODE REVIEW

File: models/recipe-model.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+const recipeSchema = new Schema({
+  title: {type: String, required: true, unique: true},
+  level: {type: String, enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]},
+  ingredient: {type: []},
```
Comment (ta-web-paris) [inline models/recipe-model.js]: You should actually write the word array here  `type: Array`

File: models/recipe-model.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    type: String,
+    enum: ["Breakfast", "Dish", "Snack", "Drink", "Dessert", "Other"]
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg"
+  },
+  duration: {type: String, min: 0},
+  creator: {type: String},
+  created: {type: [Date]}
```
Comment (ta-web-paris) [inline models/recipe-model.js]: You can set a default created time by adding `default : Date.now`

**Comment by ta-web-paris** [comment]
You seem to have understood the lesson 👏

When you're testing, it's important to console log both the response (res) and error (err) to see what type of data you're getting

File: models/recipe-model.js (hunk 2 above)
Comment (kevbarns) [reply models/recipe-model.js]: I heard nizar saying that it should be made in the code when inserting the value, but it's great news actually ! can it be done the same way with timestamp ?

**Pull request closed without merging**